code/game/gamemodes/vampire/vampire_powers/basic.dm

```diff
@@ -79,20 +80,20 @@
 		return
 
 	V.chosen_click_attack = src
-	H << "<span class='vampirewarning'>[src] is now active. (Use your middle mouse button on anyone to use.)
+	H << "<span class='vampirewarning'>[src] is now active. (Use your middle mouse button on anyone to activate.)"
 	return 1
 
-/obj/effect/proc_holder/vampire/gaze/action_on_click(/mob/living/carbon/human/H, /datum/vampire/V, atom/target)
+/obj/effect/proc_holder/vampire/gaze/action_on_click(mob/living/carbon/human/H, /datum/vampire/V, atom/target)
 	. = ..()
 	if(!.)
 		return
 
-	if(!isliving(target))
+	if(!ishuman(target))
```

Comment:
not sure if this is intended to stun borgs, if so use target.Weaken rather than stun in https://github.com/Super3222/yogstation/pull/2/files#diff-fd2581a5304afb14103bd5f8958f358fR110